script/conversion/add_links_to_measures

```diff
@@ -1,25 +1,40 @@
 #!/usr/bin/env python3
+
 import os
+from pathlib import Path
+from typing import Any, TextIO
+
+import click
 import yaml
+from yaml.loader import FullLoader, Loader, SafeLoader
+
+rename_ai_act_labels_dict = {
```

Comment:
Why do we do this?

Reply:
@ravimeijerrig

Reply:
The Loader is to maintain the order of entries in the yaml. If there is a better way to do this, let me know! 
The rename is because we use slightly different names, would be good to align this in the first place. There are a lot of changes there still on AK side.